android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -646,9 +647,12 @@ public void stopMonitoringRegion() {
         @Override
         public void run() {
 
-            LatLngBounds bounds = map.getProjection().getVisibleRegion().latLngBounds;
-            if (lastBoundsEmitted == null ||
-                    LatLngBoundsUtils.BoundsAreDifferent(bounds, lastBoundsEmitted)) {
+            Projection projection = map.getProjection();
+            VisibleRegion region = (projection != null) ? projection.getVisibleRegion() : null;
+            LatLngBounds bounds = (region != null) ? region.latLngBounds : null;
```

Comment:
nit: you don't need the parenthesis around `projection != null` and `region != null`